roboteam_ai/src/jsons/jsonBasicTestTwo.json

```diff
@@ -0,0 +1,81 @@
+{
```

Comment:
Maybe add some kind of comment somewhere that these JSONS are for testing purposes only?

Reply:
What is a nice place for this? The JSONS themselves do not have comments.